.github/workflows/ci.yaml

```diff
@@ -67,7 +73,7 @@ jobs:
   # For branch protection settings, this job provides a "stable" name that can be used to gate PR merges
   # on "all matrix jobs were successful".
   conclusion:
-    needs: test
+    needs: [test, pre-commit]
```

Comment:
@alexeagle While adding this here, I noticed that `verify-bcr-patches` and `test-release` are not listed here as deps. Is this expected?

Reply:
No, that's a (minor) bug.